frameworks/CSharp/aspnetcore/benchmark_config.json

```diff
@@ -20,9 +19,26 @@
       "notes": "",
       "versus": ""
     },
+    "middleware-json": {
+      "setup_file": "setup-json",
+      "json_url": "/json",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Micro",
+      "database": "None",
+      "framework": "aspnetcore",
+      "language": "C#",
+      "orm": "Raw",
+      "platform": "NET",
```

Comment:
And even further out "NETCoreAOT". Not sure if these are inline with other pivots of "platform" though, looking for guidance here.